services/thermal_mgr/thermal_mgr.c

```diff
@@ -44,17 +44,57 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
 
-  return ERR_CODE_SUCCESS;
+  if (xQueueSend(thermalMgrQueueHandle, (void *) &event, 5000/portTICK_PERIOD_MS) == pdPASS)
+  {
+    return ERR_CODE_SUCCESS;
+  }  
+  if (thermalMgrQueueHandle == NULL || event == NULL)
+  {
+    return ERR_CODE_INVALID_ARG; 
+  }
+
+  return ERR_CODE_UNKNOWN; 
```

Comment:
only way we would get here is if the args were valid and the queuesend did not pass which means that the queue was full. ERR_CODE_UNKNOWN should be avoided as much as possible since it does not give much info